src/garage/tf/algos/reps.py

```diff
@@ -6,14 +6,17 @@
 import scipy.optimize
 import tensorflow as tf
 
-from garage import _Default, log_performance, make_optimizer, TrajectoryBatch
+from garage import _Default, log_performance, make_optimizer
 from garage.np.algos import RLAlgorithm
 from garage.sampler import RaySampler
 from garage.tf import paths_to_tensors
 from garage.tf.misc import tensor_utils
 from garage.tf.misc.tensor_utils import flatten_inputs, graph_inputs
 from garage.tf.optimizers import LbfgsOptimizer
 
+# skip isort because isort result conflicts with yaqf auto-format
```

Comment:
the correct solution here is to disable YAPF, not isort. see several files in this same PR for an example.

unfortunately, whitespace matters.

example:
```py
"""My cool module."""
# yapf: disable
import os
import sys

import numpy as np

from garage import StepType
                                                              # <-- note the newline here
# yapf: enable
```